src/peerbook/libp2p_peerbook.erl

```diff
@@ -117,12 +119,17 @@ put(#peerbook{tid=TID, stale_time=StaleTime}=Handle, PeerList0, Prevalidated) ->
                              _ ->
                                  NewPeerId = libp2p_peer:pubkey_bin(NewPeer),
                                  case unsafe_fetch_peer(NewPeerId, Handle) of
+                                     {error, disabled} ->
```

Comment:
increment the `peer_rejected` gauge here as well or a different `peer_disabled` to get a fix on those more directly?